src/Context.php

```diff
@@ -40,6 +43,8 @@ public function __construct($args)
         // headers list for signing; not necessary for verifying.
         if (isset($args['headers'])) {
             $this->headers = $args['headers'];
+        } else {
+            $this->headers = null;
```

Comment:
this is the default value so unnecessary to set it explicitly

Reply:
Good point. e8d9b91904f43f80af78bca6cacbc72208669a06